great-idea/css/index.css

```diff
@@ -218,29 +213,36 @@ h1, h2, h3, h4, h5 {
 		}
 
 	.container-bottom {
-		border: 3px solid red;
+		/* border: 3px solid red; */
+		/* margin: 10px 40px 0px 40px; */
+		padding-top: 3px;
 		display: flex;
-		justify: space-between;
+		
+		justify-content: space-around;
 	}
 
 		.box-bottom-left {
-			border: 2px solid black;
-			/* width: 350px; */
-			padding: 7px;
-			flex-wrap: wrap;
-			align-self: center
+			/* border: 2px solid red; */
+			width: 49%;
+			padding: 10px;
+		
 		}
+			.bottom-bullets	{
+				/* border: 2px solid red; */
+				list-style: inside;	
+			}
 
 		.box-bottom-right {
-			border: 2px solid black;
-			/* width: 350px; */
+			/* border: 2px solid black; */
+			width: 49%;
 			padding: 7px;
-			flex-wrap: wrap;
-			align-self: center
+			/* flex-wrap: wrap; */
+			/* align-self: center */
 		}
 	
 	.box-footer {
-		border: 3px solid red;
+		/* border: 3px solid red; */
+		margin: 10px 40px 0px 40px;
 		display: flex;
 		justify-content: center;
 	}
```

Comment:
Your margins throughout the page need some work to match with the design file. It doesn't have to be pixel perfect, but you should be within eyeball range.